src/components/Images.vue

```diff
@@ -283,7 +283,9 @@ export default {
       this.imageOutputCuller = getImageOutputCuller(command);
 
       if (this.$refs.fullWindow) {
-        this.$refs.fullWindow.scrollTop = this.$refs.fullWindow.scrollHeight;
+        this.$nextTick(() => {
+          this.$refs.fullWindow.parentElement.parentElement.scrollTop = this.$refs.fullWindow.scrollHeight;
```

Comment:
FWIW, `this.$el.parentElement` works too (because it's the parent of the component); though maybe emitting an event would make more sense here.

Reply:
which event?